package.json

```diff
@@ -84,7 +91,7 @@
     "rncryptor-js": "0.0.1",
     "secp256k1": "3.8.0",
     "secure-random": "^1.1.2",
-    "sequelize": "5.19.3",
+    "sequelize": "^5.19.3",
```

Comment:
is this just an update for the minor version? I think the `^` just updates the minor version number automatically

Reply:
yes, npm showed me it (or a dependency of it) had a vulnerability, so i ran `npm i sequelize`, npm did this automatically. npm adds that by default, i can remove it if you want

Reply:
ohh ok gotcha that makes sense, but yea this looks good to me 

just not sure why it only added the `^` and didn't update that minor version number aswell